AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/image-preview.js

```diff
@@ -150,7 +152,7 @@ define([
                 showLoader: true,
                 data: {
                     'image_id': record.id,
-                    'limit': 4
+                    'limit': self.tabImagesLimit
```

Comment:
`this` should work here,  it is used a few lines above for setting the URL. Otherwise, it's always better to specify `context: this` or `.bind(this)` instead of declaring a variable.